heartbeat/_meta/beat.yml

```diff
@@ -1,191 +1,20 @@
 ################### Heartbeat Configuration Example #########################
 
+# This file is an example configuration file highlighting only some common options.
+# The heartbeat.full.yml file in the same directory contains all the supported options
+# with detailed comments. You can use it for reference.
+#
+# You can find the full configuration reference here:
+# https://www.elastic.co/guide/en/beats/heartbeat/index.html
+
 ############################# Heartbeat ######################################
 
 # Configure monitors
 heartbeat.monitors:
-- type: icmp # monitor type `icmp` (requires root) uses ICMP Echo Request to ping
-             # configured hosts
-
-  # Monitor name used for job name and document type.
-  #name: icmp
-
-  # Enable/Disable monitor
-  #enabled: true
-
-  # Configure task schedule using cron-like syntax
-  schedule: '*/5 * * * * * *' # exactly every 5 seconds like 10:00:00, 10:00:05, ...
+- type: icmp
```

Comment:
I think you were saying the `icmp` is not available on Windows, right? I'd like the default configuration to work by default on all platforms. Plus I find the case of HTTP monitoring more natural for Heartbeat.